app/controllers/stories/feeds_controller.rb

```diff
@@ -46,23 +47,16 @@ def signed_out_base_feed
         Articles::Feeds::Basic.new(user: nil, page: @page, tag: params[:tag]).feed
       else
         Articles::Feeds::LargeForemExperimental.new(user: current_user, page: @page, tag: params[:tag])
-          .default_home_feed(user_signed_in: user_signed_in?)
+          .default_home_feed(user_signed_in: false)
       end
     end
 
     def timeframe_feed
-      feed = Articles::Feeds::LargeForemExperimental.new(user: current_user, page: @page, tag: params[:tag])
-      feed.top_articles_by_timeframe(timeframe: params[:timeframe])
+      Articles::Feeds::Timeframe.call(params[:timeframe], tag: params[:tag], page: @page)
```

Comment:
This was independent of the rest of `LargeForemExperimental` so it became a dedicated service now.